### PR TITLE
Update cabal.project

### PR DESCRIPTION
**Pull request by ksaric**
Fix cabal file to be in sync with the latest changes.

### CODE REVIEW

**Comment by ksaric** [comment]
Ugh:
```
patching script interpreter paths in /nix/store/4689dj4s7pb4w3zkz3rvmchkx097dcms-cardano-shell-0.1.0.0-exe-node-ipc
--
  | checking for references to /build in /nix/store/4689dj4s7pb4w3zkz3rvmchkx097dcms-cardano-shell-0.1.0.0-exe-node-ipc...
  | WARNING: there are broken packages.  Run 'ghc-pkg check' for more details.
  | WARNING: there are broken packages.  Run 'ghc-pkg check' for more details.
  | WARNING: there are broken packages.  Run 'ghc-pkg check' for more details.
  | WARNING: there are broken packages.  Run 'ghc-pkg check' for more details.
  | WARNING: there are broken packages.  Run 'ghc-pkg check' for more details.
  | WARNING: there are broken packages.  Run 'ghc-pkg check' for more details.
  | shc: readCoverageData/filePath/stackProj: couldn't find cardano-shell-0.1.0.0-I7uYWDztYFECtiwByAfm0J
  | CallStack (from HasCallStack):
  | error, called at src/SHC/Coverage.hs:92:38 in stack-hpc-coveralls-0.0.4.0-DVSbvFk6SFSsyu4CFfgz7:SHC.Coverage


```

**Comment by ksaric** [comment]
Temporarily removed the coverage check using Coveralls.

File: cardano-shell.cabal [hunk 1]
```diff
@@ -70,6 +70,31 @@ library
                        -Wredundant-constraints
                        -Wpartial-fields
 
+
+
+executable cardano-shell-exe
+  main-is: Main.hs
```
Comment (dcoutts) [inline cardano-shell.cabal]: What is the purpose of this? I don't get it.

Comment (ksaric) [reply cardano-shell.cabal]: It was my impression that this was what was missing for the Coveralls to run. Removed.

**Pull request merged**